scarb-metadata/src/lib.rs

```diff
@@ -231,6 +232,9 @@ pub struct PackageMetadata {
     #[serde(flatten)]
     pub manifest_metadata: ManifestMetadata,
 
+    /// Compiler experimental features allowed for this package.
+    pub experimental_features: ExperimentalFeaturesConfig,
```

Comment:
Agree, this will also avoid a risky `cairo-lang-*` dependency in this crate.

Reply:
Also, you must use `Option<_>` here because only Scarbs since a particular release will emit this field, and this crate must have both forward and backward compatibility.